samplecode/src/main/java/com/microsoft/azure/sample/User.java

```diff
@@ -18,7 +20,9 @@
     @PartitionKey
     private String lastName;
 
-    public User(String emailAddress, String firstName, String lastName) {
+    @JsonCreator
+    public User(@JsonProperty("emailAddress") String emailAddress, @JsonProperty("firstName") String firstName,
```

Comment:
user has to specify jsonproperty one by one?...

Reply:
This is one way, or the user can create an empty constructor. With lombok annotation `@ NoArgsConstructor`